src/test/java/com/artipie/asto/S3StorageTest.java

```diff
@@ -71,6 +79,34 @@ void shouldUploadObjectWhenSave(final AmazonS3 client) throws Exception {
         MatcherAssert.assertThat(downloaded, Matchers.equalTo(data));
     }
 
+    @Test
+    void shouldFailToSaveMultipartUploadWhenFailedToReadContent(final AmazonS3 client) {
+        final String bucket = UUID.randomUUID().toString();
+        client.createBucket(bucket);
```

Comment:
@olegmoz Let's inline bucket

Reply:
@HDouss Can't do it. `bucket` is used in `client.createBucket(bucket)` first and then in `this.storage(bucket)`. What actually makes sense is to move bucket name generation and creation to test setup method, because it is done in every test. I'll do it later in other PR.